### PR TITLE
Clarified MUST requirement for JSON format

### PR DESCRIPTION
**Pull request by deissnerk**
Addresses #643

### CODE REVIEW

**Comment by duglin** [comment]
I believe this was the original intent, so LGTM

**Comment by deissnerk** [comment]
I realized that the wording is pretty much the same for AMQP, MQTT and Kafka. As MQTT and Kafka do only support structured mode for a certain protocol version, the situation is different there. Nevertheless I think the wording also fits there. 

@clemensv In the MQTT binding I removed a sentence stating that MQTT 5.0 MAY support additional formats. I think that was redundant, as the whole concept of structured mode already suggests that additional formats may be used.

File: kafka-protocol-binding.md [hunk 1]
```diff
@@ -76,16 +76,17 @@ the Kafka message's value section as-is, with the `content-type` header value
...
 Kafka message's [header section][kafka-message-header].
 
-Implementations that use Kafka 0.11.0.0 and above MAY use either _binary_
-or _structured_ modes. Implementations that use Kafka 0.10.x.x and below
-MUST use only use _structured_ mode and encode the event in JSON. This is
-because older versions of Kafka lacked support for message level headers.
+Implementations that use Kafka 0.11.0.0 and above MAY use either _binary_ or
+_structured_ modes. Implementations that use Kafka 0.10.x.x and below MUST only
+use _structured_ mode and encode the event in JSON. This is because older
+versions of Kafka lacked support for message level headers.
```
Comment (duglin) [inline kafka-protocol-binding.md:82]: just checking... this was just a formatting change, right?

**Comment by duglin** [comment]
still LGTM

File: kafka-protocol-binding.md (hunk 1 above)
Comment (deissnerk) [reply kafka-protocol-binding.md:82]: There was a typo. the word "use" occurred twice. I thought I could just fix it, while doing my edits.

Comment (duglin) [reply kafka-protocol-binding.md:82]: ah missed that - thanks!

**Comment by duglin** [comment]
Approved on 6/18 call

**Pull request merged**